packages/rocket-backup-aws-infrastructure/src/utils/point-in-time-recovery.ts

```diff
@@ -0,0 +1,7 @@
+import { Table } from '@aws-cdk/aws-dynamodb'
+
+export const applyPointInTimeRecoveryBackup = (table: Table): void => {
+  table.node['host'].table._cfnProperties.pointInTimeRecoverySpecification = {
```

Comment:
`host` is accessed as a string to avoid this syntax, since host is private on `ConstructNode`:

`(table.node as any).host.table._cfnProperties.pointInTimeRecoverySpecification = { ... }`